langlib/lang.array/src/main/ballerina/src/lang.array/Module.md

```diff
@@ -1,3 +1,5 @@
 ## Module Overview
 
-This module provides lang library list operations defined by the language specification 2019R2.
+This module provides lang library list operations defined by the language specification 2019R3. All the functions listed 
+below are supported by resizable arrays. Fixed-length arrays and tuples do not support any operations that violate the 
```

Comment:
```suggestion
in this module are supported by lists that do not have a fixed length. Fixed-length arrays and tuples do not support any operations that violate the 
```

Reply:
Isn’t it too wordy?

Reply:
It's OK IMO, and it's better to stick to spec terms right?